### PR TITLE
Basic salary details page for employees

### PR DESCRIPTION
**Pull request by abhishekkhanduri**
#1844

### Description
Added salary page
Created employee_salaries table 
Created policies for salary page
Salary encryption and decryption added

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Salary/Policies/EmployeeSalaryPolicy.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    /**
+     * Determine whether the user can view the salary.
+     *
+     * @param  \Modules\User\Entities\User  $user
+     * @return mixed
+     */
+    public function view(User $user)
+    {
+        return $user->hasRole('super-admin');
+//        return $user->hasPermissionTo('employee_salary.view');
```
Comment (GauravGusain98) [inline Modules/Salary/Policies/EmployeeSalaryPolicy.php:21]: remove this line and uncomment all the commented lines in this file

File: Modules/Salary/Routes/web.php [hunk 2]
```diff
@@ -13,10 +13,11 @@
...
 });
 
-Route::prefix('salary-settings')->middleware('auth')->group(function () {
-    Route::get('/', 'SalarySettingController@index')->name('salary-settings.index');
-    Route::post('/update/', 'SalarySettingController@update')->name('salary-settings.update');
-});
+//Route::prefix('salary-settings')->middleware('auth')->group(function () {
+//    Route::get('/', 'SalarySettingController@index')->name('salary-settings.index');
+//   Route::post('/update/', 'SalarySettingController@update')->name('salary-settings.update');
+//});
```
Comment (GauravGusain98) [inline Modules/Salary/Routes/web.php:23]: Why we have commented this?

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Minor feedbacks. Please fix

File: Modules/Salary/Routes/web.php (hunk 2 above)
Comment (abhishekkhanduri) [reply Modules/Salary/Routes/web.php:23]: Because right now we are not using these routes.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**